packages/nimble-components/src/table/specs/table-column-specs/table-column-anchor-hld.md

```diff
@@ -0,0 +1,135 @@
+# Anchor Table Column HLD
+
+## Problem Statement
+
+We need to support links in table cells. We will do this by creating a new column type.
+
+## Links To Relevant Work Items and Reference Material
+
+[GitHub issue: Table allows configuration of hyperlink column type (#1012)](https://github.com/ni/nimble/issues/1012)
+
+## Implementation / Design
+
+This is a fairly straightforward execution of our custom column type pattern. The anchor column will use two fields from the data records: one for the link's visible text (the "label") and one for the url (the "href"). The column configuration will contain the rest of the native anchor parameters: `hreflang`, `ping`, `referrerpolicy`, `rel`, `target`, `type`, and `download`. The assumption is that these should be the same for each link of a given column (if used at all). An alternative would be to have a `*FieldName` property for each of the native anchor parameters, but that seems unlikely to be useful.
+
+If a table record is missing a href value a text span will be rendered rather than a link.
+
+The column will also take an optional `placeholder` value to use when a record does not define a label.
+
+-   _Element name_: `nimble-table-column-anchor`
+-   _Attributes/properties_:
+    -   `labelFieldName`
+    -   `hrefFieldName`
+    -   `placeholder`
+    -   `hreflang`
+    -   `ping`
+    -   `referrerpolicy`
+    -   `rel`
+    -   `target`
+    -   `type`
+    -   `download`
+
+### Cell Template
+
+We will conditionally render either a `nimble-anchor` or a `span` of text. If no label is provided, but we have a URL, we'll render a link with the URL itself as the label.
```

Comment:
nit: Throughout this document, you use "url" (lowercase), expect here where it is in all caps. Can you pick one capitalization and use it consistently throughout this spec?

Reply:
Done